src/config/2019.json

```
@@ -1037,6 +1037,15 @@
       "website": "https:\/\/www.webpagetest.org\/",
       "github": "pmeenan",
       "twitter": "patmeenan"
+    },
+      "SilentJMA": {
+      "name": "JABANE Mohamed Ayoub",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https:\/\/avatars0.githubusercontent.com\/u\/16384750?s=460&v=4",
```

Minor tweak to use smaller image size and change order for consistency:

```suggestion
      "avatar_url": "https:\/\/avatars0.githubusercontent.com\/u\/16384750?v=4&s=200",
```